components/admin_console/audits/audits.jsx

```diff
@@ -51,18 +58,38 @@ export default class Audits extends React.PureComponent {
         );
     }
 
-    render() {
-        let content = null;
+    activityLogHeader = () => {
+        const style = {
+            display: 'inline-block',
+            margin: 0,
+        };
+        return (
+            <h4 style={style}>
+                <FormattedMessage
+                    id='admin.complianceMonitoring.userActivityLogsTitle'
+                    defaultMessage='User Activity Logs'
+                />
+            </h4>
+        );
+    }
 
+    renderComplianceReports = () => {
         if (!this.props.isLicensed) {
             return <div/>;
         }
+        return (
+            <ComplianceReports/>
+        );
+    }
+
+    render() {
+        let content = null;
 
         if (this.state.loadingAudits) {
             content = <LoadingScreen/>;
         } else {
             content = (
-                <div style={style.auditTable}>
+                <div style={this.auditTableStyle()}>
```

Comment:
Why change into function?  I believe this was turned into constant before so that it's always referring to the same object and will not cause unnecessary re-render.

Reply:
I failed to clean up after myself when experimenting. Removing 🗑